apps/allocations/app/components/App/Budgets.js

```diff
@@ -70,24 +62,22 @@ const Budgets = () => {
     setPanel({
       content: NewBudget,
       data: {
-        heading: 'Edit budget',
+        heading: editingBudget.amount > 0 ? 'Edit budget' : 'Reactivate budget',
```

Comment:
Rather than our entire frontend needing to know that "`amount === 0` means `inactive`", I would rather we added a line to our `script.js` logic to set `budget.active` to `true` or `false`. Then this line can become:

```suggestion
        heading: editingBudget.active ? 'Edit budget' : 'Reactivate budget',
```

Reply:
This has been updated in the latest commit